src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -56,8 +56,8 @@
 
 <Popover bind:visible anchor={button} direction="rtl">
 	<div class="flex flex-col gap-4" data-tid={NAVIGATION_MENU}>
-		{#if walletOptions}
-			<MenuWallet on:icMenuClick={hidePopover} />
+		{#if yourAddressesOption}
+			<MenuYourAddresses on:icMenuClick={hidePopover} />
```

Comment:
Out of curiosity, is there a good reason to not show this option when a user is in the settings route?

Reply:
that is a good question... I remember there was some talk about this at some point, but the PR is https://github.com/dfinity/oisy-wallet/pull/1900

Maybe worth asking design team?

Reply:
Perfect, thank you for linking this issue - It seems as the YourAdresses functionality also does not work on the /explore route, so I have to exclude it there as well.